### ISSUE TITLE
Display aircraft's route on the scope

### ISSUE THREAD

**Issue by erikquinn** [openscope/openscope#1467]
A new scope command is needed, `QU`, which will display the route of the aircraft's route of flight from their current position.

This will also map to the F6 key, as per the scope command mapping plan found [here](https://github.com/openscope/openscope/issues/14#issue-198392672).

### Requirements:
- Should leave a gap in line after a vector waypoint and whatever the next waypoint is (if any)
- Continue the vector line from the aircraft's current position all the way to the destination airport (or last fix if a departure a/c)
- Should be displayed for 15 (?) seconds after commanded
- Should be able to simultaneously display the routes for as many aircraft as the user desires
- Should clear all displayed route lines when `QU` is submitted as a command with _no_ subsequent parameters (aircraft callsign/cid/click/etc)
- Usable by manually typing `QU` in the scope command input box, or by pressing the F6 key in scope command context

**Comment by n8rzz** [comment]
>display the route of the aircraft's route of flight from their current position

I assume this Is the _*planned*_ route?  

Either way, any implementation here should be from the ground up.  Meaning: don't use the existing `show flight path` logic.  It won't be able to handle full flight plans of multiple aircraft over time.

**Comment by erikquinn** [comment]
Nope, should be MUCH more simple than how that crap works. Simple parsing of the fixes on the flightplan, and drawing a line to them. Just like how SIDs/STARs are drawn on the map in fact (give or take).
